public/js/profile.js

```diff
@@ -0,0 +1,34 @@
+$(function(){
+  $(".question-body, .question-answer, .best-answer").hide();
+    $(".show-more").on('click', function(event) {
+    event.preventDefault();
+    $(this).siblings("li, button").show()
+
+  });
+});
+
+$(function(){
+   $(".best-answer").on('click', function(event) {
+   var id = $(this).prev()
+   var answer_id = id.attr('id')
+   var user_id = $("h2").attr('id')
+   var question_id = $(this).prev().prev().prev().attr('id');
```

Comment:
This is very error prone. Consider using `$(this).closest('WHATEVERSELECTOR');` to find the thing you're looking for? It's also much more readable.